_rules/SC1-2-1-audio-transcript.md

```diff
@@ -21,26 +18,28 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or with a "play button".
+The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button".  If a play button exists the rule applies if the button is [visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
 
-A play button is an interactive element that when activated, plays the audio. The play button must be [visible on the page](#visible-on-the-page) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+A play button is an interactive element that when activated, plays the audio. The play button must be [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
The two paragraphs should be coherent. In the first, the play button needs to be exposed to assistive technologies and in the second it needs to be included in the accessibility tree.

Reply:
This may also speak to creating a stand-alone definition for "play button" (see discussion above).

Reply:
We feel that inline definition is the way to go for "play button". Since there are no strong objections we will leave the rule as is.